CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ## [x.x.x]
 ### Added
+- Add VAF to cancer case general report, and remove some of its unused fields
```

Comment:
I've just realized that the current changelog contains two recent features in the wrong position:

- Chromograph integration for displaying PNGs in case-page
- Variants filter compatible with genome browser location strings

Could you move them up here instead? I guess that is what happens if you keep a PR open for quite some time! 😄